src/libexec/zmsaupdate

```diff
@@ -54,8 +54,10 @@ if (!$rule_updates_enabled) {
   exit;
 }
 
+my $really_allow_plugins = getLocalConfig("antispam_saupdate_reallyallowplugins");
+   $really_allow_plugins = $really_allow_plugins =~ /true/ ? "--reallyallowplugins" : "";
```

Comment:
shouldn't we have `--allowplugins` in the else or is that optional and not required ?